src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+html body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  --text-active: hsl(193, 100%, 43%);
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
```

Comment:
<a> is an inline element by default, so specifying `width` and `height` on it doesn't do anything. Try turning off those styles in browser dev tools.